literalai/prompt_engineering/prompt.py

```diff
@@ -84,7 +84,7 @@ def to_dict(self) -> PromptDict:
             "id": self.id,
             "createdAt": self.created_at,
             "updatedAt": self.updated_at,
-            "type": self.type,
+            "type": self.type.value if isinstance(self.type, GenerationType) else self.type,
```

Comment:
Should the `from_dict` be updated as well?

Reply:
I think in the from_dict default being chat is fine.